cmd/mesh/upgrade.go

```diff
@@ -177,6 +181,8 @@ func upgrade(rootArgs *rootArgs, args *upgradeArgs, l *logger) (err error) {
 	l.logAndPrintf("Upgrade version check passed: %v -> %v.\n", currentVersion, targetVersion)
 
 	// Read the overridden values from args.inFilename
+	// TODO: Is this correct? Seems to be checking only the overlays under global. Other parts in ICPS can be
+	// overlaid too.
```

Comment:
@elfinhe Can you create an issue and check this part?

Reply:
I already did: https://github.com/istio/istio/issues/19135